opal/mca/btl/openib/btl_openib_component.c

```diff
@@ -2363,6 +2363,13 @@ static float get_ib_dev_distance(struct ibv_device *dev)
         goto out;
     }
 
+    opal_output_verbose(5, opal_btl_base_framework.framework_output,
+                        "hwloc_distances->nbobjs=%d", hwloc_distances->nbobjs);
```

Comment:
It might make sense to first opal_output_verbose() something that contains the ibv_get_device_name() -- i.e., so that you know what device all the subsequent opal_output_verbose()'s are referring to.

Other than that, it looks fine to me.